### ISSUE TITLE
Stop using Handles in some object names

### ISSUE THREAD

**Issue by jmarrec** [NREL/OpenStudio#3080]
This makes diffs of the IDF resulting from two subsequent runs very annoying.

Non exhaustive list of objects:

- [x] SCHEDULE:WEEK:DAILY

- [x] ZoneCapacitanceMultiplier:ResearchSpecial

- [x] LifeCycleCost:Parameters

- [x] Daylighting:Controls

- [ ] **Any others?**

Linked PR: #3081

**Comment by eringold** [comment]
Also Daylighting:Controls

**Comment by MatthewSteen** [comment]
* `LifeCycleCostParameters`

**Comment by jmarrec** [comment]
I thought `Daylighting:Controls` was a no-op but it isn't indeed @eringold.
As you said on slack `OS:Daylighting:Control` name gets mapped to `Daylighting:ReferencePoint`, but the IDF `Daylighting:Controls` which maps the zone name to the `Daylighting:ReferencePoint`s indeed has a handle for a name, so I now rename it like "#{zoneName} DaylightingControls".

**Example:**

```
m = OpenStudio::Model::exampleModel()
# Only one control in "Space 2", which belongs to "Thermal Zone 1"
dc =m.getDaylightingControls[0]
s = dc.space.get
z = s.thermalZone.get

# True
z.primaryDaylightingControl.get == dc

dc.setName("#{s.name.to_s} DaylightControl")

ft = OpenStudio::EnergyPlus::ForwardTranslator.new
w = ft.translateModel(m)

puts w.getObjectsByType("Daylighting:Controls".to_IddObjectType)
puts w.getObjectsByType("Daylighting:ReferencePoint".to_IddObjectType)
```

**Before:**

```
Daylighting:Controls,
  {c6a1c79f-67c7-4c56-9a3f-ca932d316f72}, !- Name
  Thermal Zone 1,                         !- Zone Name
  ,                                       !- Daylighting Method
  ,                                       !- Availability Schedule Name
  Continuous,                             !- Lighting Control Type
  ,                                       !- Minimum Input Power Fraction for Continuous or ContinuousOff Dimming Control
  ,                                       !- Minimum Light Output Fraction for Continuous or ContinuousOff Dimming Control
  ,                                       !- Number of Stepped Control Steps
  ,                                       !- Probability Lighting will be Reset When Needed in Manual Stepped Control
  Space 2 DaylightControl,                !- Glare Calculation Daylighting Reference Point Name
  0,                                      !- Glare Calculation Azimuth Angle of View Direction Clockwise from Zone y-Axis {deg}
  ,                                       !- Maximum Allowable Discomfort Glare Index
  ,                                       !- DElight Gridding Resolution {m2}
  Space 2 DaylightControl,                !- Daylighting Reference Point Name 1
  0.25,                                   !- Fraction of Zone Controlled by Reference Point 1
  500;                                    !- Illuminance Setpoint at Reference Point 1 {lux}

Daylighting:ReferencePoint,
  Space 2 DaylightControl,                !- Name
  Thermal Zone 1,                         !- Zone Name
  15,                                     !- X-Coordinate of Reference Point {m}
  5,                                      !- Y-Coordinate of Reference Point {m}
  1.1;                                    !- Z-Coordinate of Reference Point {m}
```

**After:**

```
Daylighting:Controls,
  Thermal Zone 1DaylightingControls,      !- Name
  Thermal Zone 1,                         !- Zone Name
  ,                                       !- Daylighting Method
  ,                                       !- Availability Schedule Name
  Continuous,                             !- Lighting Control Type
  ,                                       !- Minimum Input Power Fraction for Continuous or ContinuousOff Dimming Control
  ,                                       !- Minimum Light Output Fraction for Continuous or ContinuousOff Dimming Control
  ,                                       !- Number of Stepped Control Steps
  ,                                       !- Probability Lighting will be Reset When Needed in Manual Stepped Control
  Space 2 DaylightControl,                !- Glare Calculation Daylighting Reference Point Name
  0,                                      !- Glare Calculation Azimuth Angle of View Direction Clockwise from Zone y-Axis {deg}
  ,                                       !- Maximum Allowable Discomfort Glare Index
  ,                                       !- DElight Gridding Resolution {m2}
  Space 2 DaylightControl,                !- Daylighting Reference Point Name 1
  0.25,                                   !- Fraction of Zone Controlled by Reference Point 1
  500;                                    !- Illuminance Setpoint at Reference Point 1 {lux}

Daylighting:ReferencePoint,
  Space 2 DaylightControl,                !- Name
  Thermal Zone 1,                         !- Zone Name
  15,                                     !- X-Coordinate of Reference Point {m}
  5,                                      !- Y-Coordinate of Reference Point {m}
  1.1;                                    !- Z-Coordinate of Reference Point {m}
```

**Comment by jmarrec** [comment]
Closed in #3081
